odom_node:
  ros__parameters:
    lx: 0.15
    ly: 0.15
    wheel_radius: 0.04
    odom_frame: 'odom'
    base_frame: 'base_footprint'